projects/redis.io/package.yml

```diff
@@ -0,0 +1,47 @@
+distributable:
+  url: https://download.redis.io/releases/redis-{{ version }}.tar.gz
+  strip-components: 1
+
+versions:
+  github: redis/redis
+
+
+provides:
+  - bin/redis-server
+  - bin/redis-cli
+  - bin/redis-benchmark
+
+dependencies:
+  openssl.org: ^1
+
+build:
+  dependencies:
+    # gnu.org/make: '>=4.3'
+    tea.xyz/gx/make: '*'
+    freedesktop.org/pkg-config: '>=0.29'
+    tea.xyz/gx/cc: c99
+  script:
+    make install
+    # TODO: configure default redis.conf if necessary
+  env:
+    PREFIX: ${{prefix}}
+    BUILD_TLS: yes
+test:
+  script: |
+    redis-server --test-memory 2
+    redis-cli --version
+    redis-benchmark --version
+
+    redis-server --daemonize yes
+    redis-cli --raw SET key123 value123
+    redis_value=$(redis-cli --raw GET key123)
+    echo $redis_value
+
+    redis-cli shutdown
+
+    if [ $redis_value = "value123" ]; then
+      echo "redis works!"
+    else
+      echo "redis install doesnt work!"
+      set -e
+    fi
```

Comment:
simpler exit condition might just be `test "$redis_value" = "value123"` and let the error code tell the story.

Reply:
or, even just replace the latter half with `test "$(redis-cli --raw GET key123)" = "value123"`

Reply:
that is much cleaner. added it.